pkg/blob/nodeserver.go

```diff
@@ -676,6 +694,15 @@ func waitForMount(path string, intervel, timeout time.Duration) error {
 	}
 }
 
+func getClientID(context map[string]string) string {
```

Comment:
remove this func, use `getValueInMap(context, clientIDField)` instead, that's more common func